engine/common/src/main/java/org/enso/common/ContextFactory.java

```diff
@@ -252,7 +254,7 @@ public Context build() {
   }
 
   private static HostAccess allWithTypeMapping() {
-    return HostAccess.newBuilder()
+    return HostAccess.newBuilder(HostAccess.EXPLICIT)
```

Comment:
Starting with `EXPLICIT` and then allowing `allPublicAccess` is strange. Doesn't it create `HostAccess.PUBLIC`?

Reply:
Removed in 557bd008390bc1c03c781f9f0e85c8fc4a195171